apps/corona-banner/src/app.js

```diff
@@ -1,52 +1,96 @@
-import React, { useState, useEffect } from "react";
-import CoronaSvg from "./assets/covid-virus-icon.svg";
-import Chevron from "../../../images/chevron.svg";
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import CoronaSvg from './assets/covid-virus-icon.svg'
+import Chevron from '../../../images/chevron.svg'
+
+const Banner = ({
+  newCases,
+  hospitalised,
+  deaths,
+  vaccinated,
+  vaccinatedPercentage,
+}) => {
+
+  return (
+    <a href='/tagg/coronaviruset/'>
+      <div className='corona-container'>
+        <div className='content-container'>
+          <header className='container-header'>
+            <h1 className='banner-title'>
+              Covid-19 <br /> i Finland
+            </h1>
+            <img className='virus-image' src={CoronaSvg} alt='' />
+          </header>
+          {newCases !== null && (
+            <div className='stat'>
+              <div className='stat-value'>{newCases}</div>
+              <div className='stat-label'>nya fall</div>
+            </div>
+          )}
+          {hospitalised !== null && (
+            <div className='stat mobile-hidden'>
+              <div className='stat-value'>{hospitalised}</div>
+              <div className='stat-label'>på sjukhus</div>
+            </div>
+          )}
+          {deaths !== null && (
+            <div className='stat'>
+              <div className='stat-value'>{deaths}</div>
+              <div className='stat-label'>dödsfall</div>
+            </div>
+          )}
+          {vaccinated !== null && (
+            <div className='stat'>
+              <div className='stat-value'>
+                {vaccinated}{' '}
+                <span className='stat-percent'>({vaccinatedPercentage}%)</span>
+              </div>
+              <div className='stat-label'>vaccinerade</div>
+            </div>
+          )}
+        </div>
+        <div className='chevron-container'>
+          <img className='chevron-right' src={Chevron} alt='' />
+        </div>
+      </div>
+    </a>
+  )
+}
 
 export default function App() {
-  const [ value1, setValue1] = useState(null)
-  const [ value2, setValue2] = useState(null)
-  const [ value3, setValue3] = useState(null)
-  const [ value4, setValue4] = useState(null)
+  const [newCases, setNewCases] = useState(null)
+  const [hospitalised, setHospitalised] = useState(null)
+  const [deaths, setDeaths] = useState(null)
+  const [vaccinated, setVaccinated] = useState(null)
+  const [vaccinatedPercentage, setVaccinatedPercentage] = useState(null)
+  const [isLoaded, setIsLoaded] = useState(false)
 
   useEffect(() => {
     console.log('effect')
-    fetch("https://cat-fact.herokuapp.com/facts")
-    .then(res => console.log(res))
-    
+    axios
+      .get(' https://cdn.ksfmedia.fi/corona-banner/stats.json')
+      .then(response => {
+        console.log('promise fulfilled')
```

Comment:
Let's remove these two `console.logs`, as I suppose they are for debugging purposes (not really useful in production)